lucida/commandcenter/app.py

```diff
@@ -27,20 +28,40 @@
 
 # Session.
 app.secret_key = 'A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
-	
+
 def flask_listener():
-	app.run(host='0.0.0.0', port=3000, debug=True, use_reloader=False,
-			threaded=True) 
-	
+
+    # For https
+    if os.environ.get('SECURE_HOST'):
+        print 'Starting secure flask'
+        app.run(host='0.0.0.0', port=3000, debug=True, use_reloader=False,
+                threaded=True, ssl_context=('certs/server.crt', 'certs/server.key'))
+    # For http
+    else:
+        print 'Starting non-secure flask'
```

Comment:
I'm adding a comment within this file but also I think it'll be more clear once I write out the instructions on how to start servers with and without ASR.